docs/user/unsupported/mbstyle/index.rst

```diff
@@ -29,4 +35,4 @@ Internally the extension provides greater access to the parsed style:
     List<MBLayer> layers = mbstyle.layers( "mapbox://mapbox.mapbox-streets-v6", "water" ); 
 
     //Which can be used to Generate Styler Layer Descriptor:
-    StyleLayerDescriptor sld = MBStyleTransformer.transform( layers );
\ No newline at end of file
+    FeatureTypeStyle fts = layer.transform( style );
```

Comment:
We should change this api to be a List<FeatureTypeStyle> before release, because we know fill-extrusion will need the api change.